lib/IRGen/IRGenMangler.h

```diff
@@ -86,6 +86,8 @@ class IRGenMangler : public Mangle::ASTMangler {
     beginMangling();
     appendEntity(func);
     appendOperator("Tq");
+    if (func->isDistributedThunk())
+      appendSymbolKind(SymbolKind::DistributedThunk);
     return finalize();
```

Comment:
We have to suffix it like this, earlier, in the entity or before Tq end up causing demangling issues and this seems quite fine tbh.